pod-operation/src/main.rs

```diff
@@ -11,32 +11,37 @@ use crate::components::lim_temperature::LimTemperature;
 use crate::components::pressure_transducer::PressureTransducer;
 use crate::components::signal_light::SignalLight;
 use crate::state_machine::StateMachine;
+use crate::components::hall_effect::HallEffect;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
 	tracing::subscriber::set_global_default(FmtSubscriber::default())?;
 
 	let (layer, io) = SocketIo::new_layer();
 
-	let signal_light = SignalLight::new();
-	tokio::spawn(demo::blink(signal_light));
+	// let signal_light = SignalLight::new();
+	// tokio::spawn(demo::blink(signal_light));
```

Comment:
Chore [repeating]: as noted in previous PRs, please avoid staging personal testing changes.